recipes/keras-tuner/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "keras-tuner" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f018ded5127e7c588bd78faa753fa1f4108f1f28c0e5694a06bfb1f08204215b"
+
+build:
```

Comment:
this looks like it can be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Also the library appears to be python3.6 and above only. Assuming this is going to be `noarch: python`, change `python` to `python  >=3.6` under both `host` and `run`.